File: app/src/amazon/java/org/tasks/location/GeofenceTransitionsIntentService.java
package org.tasks.location;

import android.app.IntentService;
import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import androidx.annotation.Nullable;

public class GeofenceTransitionsIntentService extends IntentService {

  public GeofenceTransitionsIntentService() {
    super("GeofenceTransitionsIntentService");
  }

  @Override
  protected void onHandleIntent(@Nullable Intent intent) {}

  public static class Broadcast extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {}
  }
}

File: app/src/generic/java/org/tasks/location/GeofenceTransitionsIntentService.kt
package org.tasks.location

import android.app.IntentService
import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class GeofenceTransitionsIntentService : IntentService("GeofenceTransitionsIntentService") {
    override fun onHandleIntent(intent: Intent?) {}
    class Broadcast : BroadcastReceiver() {
        override fun onReceive(context: Context, intent: Intent) {}
    }
}